JobRunner/JobRunner.py

```diff
@@ -436,7 +436,7 @@ def callback(self, job_params=None):
 
         self.prov = Provenance(job_params)
         self._init_workdir()
-        job_dir = self.workdir
+        job_dir = os.path.join(self.workdir, "workdir")  # TODO should handle this in mr
         # TODO: This is calling a private method.
         self.mr._init_workdir(config, job_dir, job_params)
```

Comment:
... I'm not totally sure this line is needed. I think _init_workdir_ is called internally whenever a container is run. Not convinced enough to remove it though